README.md

```diff
@@ -0,0 +1,22 @@
+# Budabot (Python rewrite)
+
+This is a rewrite of the original [Budabot](https://github.com/Budabot/Budabot) in Python3.  
+
+Budabot is an in-game chat-bot for the MMORPG Anarchy Online released by Funcom in 2001.  
+
+## Installation
+In its current state, this project is not meant to be used, other than during development.  
+
+# Developement state  
```

Comment:
I don't want to list the functionality since it is continually changing and I won't be updating this for every module I convert